package.json

```diff
@@ -16,29 +16,30 @@
 		"test": "eslint \"**/*.{js,jsx}\" && jest"
 	},
 	"dependencies": {
-		"@babel/core": "^7.15.0",
-		"@babel/plugin-proposal-class-properties": "^7.14.5",
-		"@babel/plugin-transform-react-jsx": "^7.14.9",
-		"@babel/preset-env": "^7.15.0",
-		"@babel/preset-typescript": "^7.15.0",
-		"babel-jest": "^27.0.6",
+		"@babel/core": "^7.20.7",
+		"@babel/plugin-proposal-class-properties": "^7.18.6",
+		"@babel/plugin-transform-react-jsx": "^7.20.7",
+		"@babel/preset-env": "^7.20.2",
+		"@babel/preset-typescript": "^7.18.6",
+		"babel-jest": "^29.3.1",
 		"identity-obj-proxy": "^3.0.0",
 		"isomorphic-unfetch": "^3.1.0",
-		"jest-watch-typeahead": "^0.6.4"
+		"jest-watch-typeahead": "^2.2.1"
 	},
 	"devDependencies": {
-		"@types/jest": "^27.0.0",
-		"babel-eslint": "^10.1.0",
-		"eslint": "^7.16.0",
-		"husky": "^4.2.5",
-		"jest": "^27.0.6",
-		"lint-staged": "^10.2.10",
-		"preact": "10.4.4",
-		"preact-render-to-string": "5.1.9",
-		"prettier": "^2.2.1"
+		"@babel/eslint-parser": "^7.19.1",
+		"@types/jest": "^29.2.4",
+		"eslint": "^8.30.0",
+		"husky": "^8.0.2",
+		"jest": "^29.3.1",
+		"jest-environment-jsdom": "^29.3.1",
+		"lint-staged": "^13.1.0",
+		"preact": "10.11.3",
+		"preact-render-to-string": "5.2.6",
+		"prettier": "^2.8.1"
```

Comment:
Is there a need for any of these to change (beyond Jest stuff, of course)?

Keeping the diff contained to the PR purpose is generally appreciated.